app.py

```diff
@@ -109,63 +171,46 @@ def _print_version(ctx, _, value):
+        only_solver_documents, only_analysis_documents, metrics_pushgateway_url):
+    """Sync analyses, inspection and solver results to the graph database."""
+    if verbose:
+        _LOGGER.setLevel(logging.DEBUG)
     _LOGGER.debug('Debug mode is on')
 
     _THOTH_METRICS_PUSHGATEWAY_URL = os.getenv('THOTH_METRICS_PUSHGATEWAY_URL')
 
-    graph = GraphDatabase(hosts=graph_hosts, port=graph_port)
-    graph.connect()
-
-    solver_store = SolverResultsStore(host=solver_results_store_host)
-    solver_store.connect()
+    if not only_one_kind and document_ids:
+        _LOGGER.error(
+            "Explicitly specified documents to be synced can be specified only with one of the --only-* options"
+        )
+        return 2
 
     with _METRIC_SECONDS.time():
-        for document_id, document in solver_store.iterate_results():
-            _METRIC_SOLVER_RESULTS_PROCESSED.inc()
-
-            if force_solver_results_sync or not graph.solver_records_exist(document):
-                _LOGGER.info(
-                    f"Syncing solver document from {solver_store.ceph.host} "
-                    f"with id {document_id!r} to graph {graph.hosts}"
-                )
-                try:
-                    graph.sync_solver_result(document)
-                    _METRIC_SOLVER_RESULTS_SYNCED.inc()
-                except Exception:
-                    _LOGGER.exception(
-                        "Failed to sync solver result with document id %r", document_id)
-                    _METRIC_SOLVER_RESULTS_FAILED.inc()
-            else:
-                _LOGGER.info(
-                    f"Sync of solver document with id {document_id!r} skipped - already synced")
-                _METRIC_SOLVER_RESULTS_SKIPPED.inc()
-
-        analysis_store = AnalysisResultsStore(host=analysis_results_store_host)
-        analysis_store.connect()
-        for document_id, document in analysis_store.iterate_results():
-            _METRIC_ANALYSIS_RESULTS_PROCESSED.inc()
-
-            if force_analysis_results_sync or not graph.analysis_records_exist(document):
-                _LOGGER.info(
-                    f"Syncing analysis document from {analysis_store.ceph.host} "
-                    f"with id {document_id!r} to graph {graph.hosts}"
-                )
-                try:
-                    graph.sync_analysis_result(document)
-                    _METRIC_ANALYSIS_RESULTS_SYNCED.inc()
-                except Exception:
-                    _LOGGER.exception(
-                        "Failed to sync analysis result with document id %r", document_id)
-                    _METRIC_ANALYSIS_RESULTS_FAILED.inc()
-            else:
-                _LOGGER.info(
-                    f"Sync of analysis document with id {document_id!r} skipped - already synced")
-                _METRIC_ANALYSIS_RESULTS_SKIPPED.inc()
+        if not only_one_kind or only_solver_documents:
+            _METRIC_SOLVER_RESULTS_PROCESSED, \
```

Comment:
so there will be no difference between all these metrics? what is the method signature of sync_solver_documents ?

Reply:
This way they can be used in jupyter notebook - https://github.com/thoth-station/storages/pull/293/files